swagger_server/controllers/ciupload_controller.py

```diff
@@ -0,0 +1,95 @@
+import connexion
+from datetime import date, datetime
+from typing import List, Dict
+from six import iteritems
+from ..util import deserialize_date, deserialize_datetime
+
+
+def activate_id_put(id):
+    """
+    Activate batch
+    Mark all items in the batch as available
+    :param id: Collection exercise identifier
+    :type id: str
+
+    :rtype: None
+    """
+    return 'do some magic!'
```

Comment:
Not without recompiling the code generator and changing the templates we're using. We probably will get to doing this, but not at this stage.